tests/tests/src/preserve_jsx_test.mjs

```diff
@@ -50,43 +60,63 @@ let baseProps = {
 
 let newrecord = {...baseProps};
 
-let _unary_element_with_spread_props = <input {...newrecord}  type={"text"}/>;
+let _unary_element_with_spread_props = <input {...newrecord} 
+  type={"text"}
+/>;
 
 let newrecord$1 = {...baseProps};
 
-let _container_with_spread_props = <div {...newrecord$1}  title={"barry"} className={"barry"}>
-{"Hello, world!"}
-<input type={"text"}/>
+let _container_with_spread_props = <div {...newrecord$1} 
+  title={"barry"}
+  className={"barry"}
+>
+  {"Hello, world!"}
+  <input
+    type={"text"}
+  />
 </div>;
 
 let baseChildren = [
   <span>
-  {"Hello, world!"}
+    {"Hello, world!"}
   </span>,
   <span>
-  {"Hello, world!"}
+    {"Hello, world!"}
   </span>
 ];
 
-let _container_with_spread_children = <div className={"barry"} title={"barry"}>
-{baseChildren}
+let _container_with_spread_children = <div
```

Comment:
I wonder if we could/should add a line break (+ indent) before the opening tag, too?